### PR TITLE
add php-zip support on php and php.ng

### PR DESCRIPTION
**Pull request by aboe76**
Added missing php-zip extension.

### CODE REVIEW

**Comment by gravyboat** [comment]
Thanks.

File: php/map.jinja [hunk 1]
```diff
@@ -64,6 +65,7 @@
         'ldap_pkg': 'php5-ldap',
         'php_ini': '/etc/php5/apache2/php.ini',
         'dev_pkg': 'php5-dev',
+        'dev_pkg': 'php5',
```
Comment (babilen) [inline php/map.jinja]: This breaks installations on Debian as `dev_pkg` should be `php5-dev` not `php5`. I believe that this was an unintentional leftover.

Comment (aboe76) [inline php/map.jinja]: I'm sorry

Comment (babilen) [inline php/map.jinja]: It's okay, I fixed it in https://github.com/saltstack-formulas/php-formula/pull/106

**Pull request merged**